docs/docs/reference/migration.md

```diff
@@ -3,7 +3,19 @@
 This document describes breaking changes and migrations between Constellation releases.
 Use [`constellation config migrate`](./cli.md#constellation-config-migrate) to automatically update an old config file to a new format.
 
-## Migrating from Azure's service principal authentication to managed identity authentication
+
+## Migrations to v2.19.0
+
+### Azure
+
+* To allow seamless upgrades on Azure when Kubernetes services of type `LoadBalancer` are deployed, the target  
+ load balancer in which the `cloud-controller-maanger` created the load balancing rules was changed. Instead of using the load balancer, 
```

Comment:
```suggestion
 load balancer in which the `cloud-controller-maanger` creates load balancing rules was changed. Instead of using the load balancer
```